src/views/CountryThreeWProjects/index.tsx

```diff
@@ -0,0 +1,515 @@
+import {
+    useMemo,
+    useState,
+    useCallback,
+    useContext,
+} from 'react';
+import {
+    isNotDefined,
+    listToGroupList,
+    mapToList,
+    unique,
+} from '@togglecorp/fujs';
+import {
+    generatePath,
+    useOutletContext,
+} from 'react-router-dom';
+import {
+    PencilFillIcon,
+    DownloadFillIcon,
+} from '@ifrc-go/icons';
+
+import Table from '#components/Table';
+import {
+    createStringColumn,
+    createNumberColumn,
+    createActionColumn,
+} from '#components/Table/ColumnShortcuts';
+import Container from '#components/Container';
+import Link from '#components/Link';
+import Button from '#components/Button';
+import PieChart from '#components/PieChart';
+import BlockLoading from '#components/BlockLoading';
+import KeyFigure from '#components/KeyFigure';
+import { useRequest } from '#utils/restRequest';
+import { sumSafe, denormalizeList } from '#utils/common';
+import type { CountryOutletContext } from '#utils/outletContext';
+import useTranslation from '#hooks/useTranslation';
+import ExpandableContainer from '#components/ExpandableContainer';
+import { resolveToString } from '#utils/translation';
+import {
+    numericValueSelector,
+    stringLabelSelector,
+    numericIdSelector,
+} from '#utils/selectors';
+import type { GoApiResponse } from '#utils/restRequest';
+import { type components } from '#generated/types';
+import RouteContext from '#contexts/route';
+
+import ProjectActions from './ProjectActions';
+import Map from './Map';
+import Filter, { FilterValue } from './Filters';
+
+import i18n from './i18n.json';
+import styles from './styles.module.css';
+
+interface LabelValue {
+    label: string;
+    value: number;
+}
+
+type District = NonNullable<GoApiResponse<'/api/v2/district/'>['results']>[number];
+type Project = NonNullable<GoApiResponse<'/api/v2/project/'>['results']>[number];
+
+type ProjectKey = keyof Project;
+
+// const PROJECT_STATUS_COMPLETED = 2;
+const PROJECT_STATUS_ONGOING = 1 satisfies components['schemas']['Status1d2Enum'];
+// const PROJECT_STATUS_PLANNED = 0;
+
+const emptyDistrictList: District[] = [];
+const emptyProjectList: Project[] = [];
+
+const primaryRedColorShades = [
+    'var(--go-ui-color-red-90)',
+    'var(--go-ui-color-red-60)',
+    'var(--go-ui-color-red-40)',
+    'var(--go-ui-color-red-20)',
+    'var(--go-ui-color-red-10)',
+];
+
+function filterProjects(
```

Comment:
Doesn't the server support filtering?

Reply:
Looks like we are fetching all the data and doing the aggregations client side.

cc: @frozenhelium @AdityaKhatri

Reply:
The data is being used in the map.

Reply:
@tnagorra @frozenhelium let's add a task to filter projects in server to backlog and work on it later.

Reply:
I think it cannot be done. At least for now.